app/src/main/java/com/vci/vectorcamapp/imaging/presentation/ImagingViewModel.kt

```diff
@@ -210,7 +211,10 @@ class ImagingViewModel @Inject constructor(
                                         species = species.label,
                                         sex = sex?.label,
                                         abdomenStatus = abdomenStatus?.label,
-                                    ), currentImage = bitmap
+                                    ),
+                                    currentImage = bitmap,
+                                    captureBoundingBoxUi = it.previewBoundingBoxUi,
```

Comment:
Unfortunately this did not work for me, especially if I increase the frame rate in `SpecimenImageAnalyzer`. If you click the capture button and immediately move the device to the side, the issue still persists> I think you will need to extract the detection logic out of the `inferenceRepository.classify` function. The bounding box which you get during that frame should be set to the `captureBoundingBoxUi`.